File: src/logger.ts
export class Logger {
 
 
    public static log(msg: string): void {
        console.log(msg);
    }

    public static error(msg: string): void {
        console.error("[Error]" , msg);
    }

    public static success(msg: string): void {
        console.log("[Success]" , msg);
    }

    public static warning(msg: string): void {
        console.warn("[Warning]",msg);
    }

    public static title(msg: string): void {
        console.log("\n--------------------------------------------------------------------------------");
        console.log(`    ${msg}                                                 `);
        console.log("--------------------------------------------------------------------------------");
    }


}

File: Logger/Logger.go
package Logger

import (
	"fmt"
	"os"
	"strings"

	"github.com/withmandala/go-log"
)

const titleCharNum = 67
const titleChar = "-"

var Logger = log.New(os.Stderr)

func ShowLogAppTitle() {
	Line()
	fmt.Println(`
		Github backup script by @akrck02																		
	`)
	Line()
}

func Log(msg string) {
	Logger.Info(msg)
}

func Error(msg string) {
	Logger.Error(msg)
}

func Info(msg string) {
	Logger.Info(msg)
}

func Jump() {
	Log("")
}

func Line() {
	fmt.Println(strings.Repeat(titleChar, titleCharNum))
}

func Title(title string) {
	Jump()
	Line()
	fmt.Println("   " + title)
	Line()
}
